packages/picasso/src/ButtonGroupItem/styles.ts

```diff
@@ -0,0 +1,17 @@
+import { createStyles, Theme } from '@material-ui/core/styles'
+
+import { createOutlineCommons, activeGroup } from '../Button/styles'
+
+export default (theme: Theme) =>
+  createStyles({
+    root: {
+      '&$group': {
+        ...createOutlineCommons(theme),
+        '&:active, &$active': activeGroup(theme)
+      }
+    },
+    active: {},
+    hover: {},
+    focused: {},
+    group: {}
```

Comment:
Why do you need `group` class if it's always applied to the component? Can you just add the styles to `root`?


```suggestion
  createStyles({
    root: {
      ...createOutlineCommons(theme),
      '&:active, &$active': activeGroup(theme)
    },
    active: {},
    hover: {},
    focused: {}
```

Reply:
that was my first choice too, but it didn't work, the selector wasn't specific enough and was overwritten by the default behavior of the secondary button.